mmv1/third_party/terraform/services/logging/resource_logging_project_bucket_config.go

```diff
@@ -106,6 +106,35 @@ See [Enabling CMEK for Logging Buckets](https://cloud.google.com/logging/docs/ro
 			},
 		},
 	},
+	"index_configs": &schema.Schema{
+		Type:        schema.TypeSet,
+		MaxItems:    20,
+		Optional:    true,
+		Description: `A list of indexed fields and related configuration data.`,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"field_path": &schema.Schema{
+					Type:        schema.TypeString,
+					Required: true,
+					Description: `The LogEntry field path to index.`,
+				},
+				"type": &schema.Schema{
+					Type:     schema.TypeString,
+					Required: true,
+					Description: `The type of data in this index
+Note that some paths are automatically indexed, and other paths are not eligible for indexing. See [indexing documentation]( https://cloud.google.com/logging/docs/view/advanced-queries#indexed-fields) for details.
+For example: jsonPayload.request.status`,
+				},
+				"create_time": &schema.Schema{
```

Comment:
Ah- same as https://github.com/GoogleCloudPlatform/magic-modules/pull/9109#discussion_r1343168254, `Computed` values and `TypeSet` don't play nicely. We should remove this as well (and any associated expander/flattener code, but I think those are already updated since they were shared)